### PR TITLE
BREAKING CHANGE: modified function signature and added tests

### PR DESCRIPTION
**Pull request by chribben**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,5 +4,5 @@ Tests a node server firing generated requests (using its Swagger spec) at it
 ## Usage
 ```javascript
 import { testServer } from '@springworks/server-tester';
-testServer('http://locahost:3001/', swagger_spec);
+testServer('http://locahost', 3001, swagger_spec);
```
Comment (nimerritt) [inline README.md]: Why the breaking change?

Comment (chribben) [inline README.md]: I just think it's a nicer interface to pass the host and port separated

Comment (nimerritt) [inline README.md]: Shouldn't this be an object now? {host: ..., port: 3000, ...}

File: test-util/api-builder.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
+export function ApiBuilder(api) {
```
Comment (nimerritt) [inline test-util/api-builder.js]: Is this copied in from the request-baker project? Let's make a note to extract this into another package. Hapi-sov would have use for it as well :)

**Comment by chribben** [comment]
Yes, I changed it (again) to take `base_url` instead of `host` since it expected that the scheme is supplied in the argument

**Pull request merged**